src/LowLevelMotionController.cpp

```diff
@@ -7,10 +7,41 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #include <ros/ros.h>
+
 #include "iarc7_motion/AccelerationPlanner.hpp"
 #include "iarc7_motion/QuadVelocityController.hpp"
+#include "iarc7_motion/QuadTwistRequestLimiter.hpp"
+
+#include "geometry_msgs/Twist.h"
+#include "geometry_msgs/TwistStamped.h"
 
 using namespace Iarc7Motion;
+using geometry_msgs::TwistStamped;
+using geometry_msgs::Twist;
+
+// This is a helper function that will limit a uav command using the twist limiter
+// It will eventually no longer be needed when the quad velocity controller is converted to use twists
+void limitUavCommand(QuadTwistRequestLimiter& limiter, iarc7_msgs::OrientationThrottleStamped& uav_command)
```

Comment:
Well it isn't really either, because it has force (which would technically be a wrench, I think) and orientation, which would be a quaternion